Sources/SourceControl/GitRepository.swift

```diff
@@ -414,9 +414,7 @@ public class GitRepository: Repository, WorkingCheckout {
             let result = try Process.popen(arguments: argsWithSh)
             let output = try result.output.dematerialize()
 
-            let outputs: [String] = output.split(separator: 0).map(Array.init).map({ (bytes: [Int8]) -> String in
-                return String(cString: bytes + [0])
-            })
+            let outputs: [String] = output.split(separator: 0).map{ String(decoding: $0, as: Unicode.UTF8.self) }
```

Comment:
nit: [The Rule of Kevin](https://ericasadun.com/2015/11/17/a-handful-of-swift-style-rules-swiftlang/)

```swift
let outputs: [String] = output.split(separator: 0).map({ String(decoding: $0, as: Unicode.UTF8.self) })
```